src/components/index.js

```diff
@@ -21,3 +21,4 @@ export * from './JdsPagination'
 export * from './JdsCard'
 export * from './JdsSimpleTable'
 export * from './JdsSectionMessage'
+export * from './JdsDataTable'
```

Comment:
Hi, please add a newline at the EOF :smile:

Reply:
Hello @adrianpdm , I fixed it [here](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/60/commits/f6b5f0e413ddc1e988ead877f38c81565fe332e3) thankyou